aws/aws-parallelcluster-cookbook
# Cleanup root folder of the cookbook repo

### Description of changes
* Move awsbatch_config_spec.rb to awsbatch cookbook
* Move slurm Inspec tests to slurm cookbook
* Add tag:config to some slurm tests
* Move config_bridge_network_interface_presence test into dcv
* Move ssh client check into ssh_spec.rb
* Move jq, mpich and fftw tests into install_packages_spec.rb
* Move test to verify dcv changes into dcv spec
* Move check for clusterstatusmgtd into computefleet cookbook
* Move root volume free space check on intel_hpc_spec
* Remove test/recipes folder
* Remove old system_tests
* Remove kitchen.recipes-config.yml
* Move kitchen utilities files into a kitchen folder
* Split dcv config inspec test
* Add slurm cookbook to fix-inspec-deps script
* Do not execute systemd tests on docker
* Fix dcv_services inspec tests condition
* Remove unused dcv node attribute
* Fix dcv_installed function
* Skip nvidia and gdrcopy installation and configuration on docker

### Tests

Executed:
* `bash kitchen.ec2.sh environment-config test raid-mount-ubuntu22` to verify hooks and tests are working as expected
* `bash kitchen.ec2.sh platform-config verify dcv-alinux2` to verify dcv conditions are working as expected
